static/js/util/sanctuary_test.js

```diff
@@ -4,10 +4,13 @@ import { assert } from 'chai';
 
 import { allJust, mstr } from './sanctuary';
 
-describe('sanctuary util functions', () => {
-  let maybeEquality = (j1, j2) => assert(j1.equals(j2), "just equality");
+export const maybeEquality = (j1: Maybe, j2: Maybe) => {
+  assert(j1.equals(j2), "just equality");
+};
+
+export const isNothing = (m: Maybe) => assert(m.isNothing, "should be nothing");
```

Comment:
good refactor here. the naming and messaging could be clearer, though. 

1. i think it would help to put 'assert' at the beginning of both of these functions since they exist to make an assertion. makes the test cases a bit clearer when you see the invocation of something called `assert*`
1. instead of `maybeEquality`, maybe something like `assertJustValuesEqual`? 
1. `isNothing ` --> `assertMaybeIsNothing`? 
1. instead of "just equality", maybe something like "Just values should be equal". maybe something as simple as  "should be equal" would do here